libs/ui/src/test_mode.tsx

```diff
@@ -44,8 +41,45 @@ export function TestModeCallout({
             }}
           />{' '}
           Test Ballot Mode
-        </TestModeCard>
+        </TestModeCardTouch>
       </ThemeProvider>
     </TextOnly>
   );
 }
+
+const TestModeCardDesktop = styled(Card)`
+  font-size: ${(p) => p.theme.sizes.headingsRem.h3}rem;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+
+  > div {
+    padding: 0.5rem 1rem;
+  }
+
+  flex-shrink: 0;
+`;
+
+function TestModeCalloutDesktop({
+  style,
+}: {
+  style?: React.CSSProperties;
+}): JSX.Element {
+  return (
+    <TestModeCardDesktop style={style} color="warning">
+      <Icons.Warning color="warning" /> Test Ballot Mode
+    </TestModeCardDesktop>
+  );
+}
+
+export function TestModeCallout({
+  forDesktop,
+  style,
+}: {
+  forDesktop?: boolean;
```

Comment:
What do you think about making this an enum/string literal union type instead of boolean (eg. `viewMode: 'desktop' | 'tablet'`) to be explicit about what the non-desktop alternative is?